rosa/terraform/network.tf

```diff
@@ -0,0 +1,244 @@
+# SPDX-FileCopyrightText: 2024 SAP edge team
+# SPDX-License-Identifier: Apache-2.0
+
+# Data source to get availability zones
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+# VPC
+resource "aws_vpc" "main" {
```

Comment:
Pls try the terrforam destroy and make sure the resource can be destroyed correctly.